ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml

```diff
@@ -1080,7 +1080,8 @@ tests:
     cluster_profile: azure-marketplace-qe
     env:
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
-      E2E_RUN_TAGS: '@azure-ipi and @network-openshiftsdn and not @fips'
+      E2E_RUN_TAGS: '@azure-ipi and @disconnected and @network-openshiftsdn and not
```

Comment:
Why this job need to run `disconnected` test case?

Reply:
I saw previous azure-ipi-private job includes this tag, here `@disconnected` means cluster running in a restricted network? if so, might be removed.

Reply:
> here @disconnected means cluster running in a restricted network
I think so, I guess we need to remove "@disconnected"

Reply:
@liangxia can you help us confirm that?

Reply:
Generally, we only need tag disconnected on disconnected clusters.

Reply:
thanks for the confirmation, removed tag "@disconnected"